.github/workflows/testbuild.yml

```diff
@@ -1,20 +1,14 @@
 ---
 name: Test Build
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+on: [pull_request]  # yamllint disable-line rule:truthy
```

Comment:
I disagree with this. We want to do test builds after the merge because you could also merge stuff without doing a rebase first. Config builds are fast now anyway, so let's always do them.

Reply:
Sounds good to me, i just reverted my initial commit where i added wikiupdater here, but i will add it back.